Exercises/strings/strings.js

```diff
@@ -0,0 +1,34 @@
+/*eslint-disable no-console, prefer-template */
+//1. Assign string with unicode to a variable.
+const unicodeString = '\u{1F60D}';
+console.log(unicodeString);
+//2. Declare a variable with string with escaped symbols.
+const chapterStructure = 'Title \nFamous Author Quote \' and notes \\';
+console.log(chapterStructure);
+//3. Declare a variable which will contain name of your favourite animal.
+const myAnimal = 'Puppy';
+//4. Log a message `My favourite animal is ` with dynamic variable from previous task.
+console.log(`My favourite animal is ${myAnimal}`);
+//5. Declare a variable which contain name and surname of your favourite writer in lowercase. Log his/her name with capitals first letters.
+const author = { firstName: 'michael', lastName: 'crichton' };
+function paramCapitalized(param) {
+  const paramCapitalized = param.charAt(0).toUpperCase() + param.slice(1);
+  return paramCapitalized;
+}
+console.log(paramCapitalized(author.firstName) + ' ' + paramCapitalized(author.lastName));
+//6. Declare a function which will receive password string. Check is password strong enough:
+// - should contain at least one capital letter;
+// - length no less than 8 characters.
+// - should contain at least one special character.
+const password = 'wA23422ie';
+function isPasswordStrong(passwordValue) {
+    if (passwordValue.match(/[A-Z]/)
+      && passwordValue.length >= 8
```

Comment:
is it ok by linter? looks a bit weird, I mean new lines. Also better to do check on length first, as it faster then regex, and if it not passed there is not sense to check others